torch/distributed/fsdp/_utils.py

```diff
@@ -1,21 +0,0 @@
-import torch
-
-from torch.utils._mode_utils import no_dispatch
-
-
-def _same_storage(x: torch.Tensor, y: torch.Tensor) -> bool:
```

Comment:
What happened to this function?

Reply:
I recall it might not be used anywhere, but maybe we just keep it for this PR?

Reply:
Yeah, it is not used anywhere. Why are we keeping dead code?

Reply:
It is not a big deal for this one probably, but users tend to still import private functions. It is a good practice to check that no one internally is importing it. I was just thinking to separate it into a different PR to isolate the risk of reverting.

Feel free to keep your PR as is.